### ISSUE TITLE
After posting a reply, keyboard focus is not restored to the toot I am replying to.

### ISSUE THREAD

**Issue by MarcoZehe** [nolanlawson/pinafore#42]
To reproduce, with VoiceOver and Safari on the Mac:

1. Find a toot to reply to, then navigate to the Reply button and hit Ctrl+Option+Space (keystroke to activate current control).
2. Navigate forward (ctrl+option+right arrow) to the input to compose your reply. Write something.
3. Forward some more to the Toot button and hit Ctrl+Option+Space.

Expected: Toot is posted, and focus is restored to the Reply button of the toot I was replying to.
Actual: Toot is posted, but focus ends up somewhere random. So far, I've either landed at the very bottom of the page or at the toot controls at the top that are unrelated to replying.

**Comment by nolanlawson** [comment]
Thanks very much for the bug report! I agree the focus should be restored to the reply button.
